[Bug]: Fails to compile.  FATAL: Unwanted UTF-8 character U+200B in sequence

### Version of OpenTTD

22e525dafef6273fad35c4358b942eb19ee94034

### Expected result

Should compile without error.

### Actual result

Error during compile:
```
[354/845] Compiling language traditional_chinese
FAILED: lang/traditional_chinese.lng /home/builder/build/src/build/lang/traditional_chinese.lng 
cd /home/builder/build/src/build/src/lang && /usr/bin/cmake -E make_directory /home/builder/build/src/build/lang && /home/builder/build/src/build/src/strgen/strgen -s /home/builder/build/src/openttd-git/src/lang -d /home/builder/build/src/build/lang /home/builder/build/src/openttd-git/src/lang/traditional_chinese.txt
/home/builder/build/src/openttd-git/src/lang/traditional_chinese.txt:3311: FATAL: Unwanted UTF-8 character U+200B in sequence '{BLACK}選擇遊戲版圖內最高峰的海拔，​​以海平面為基準計算'
[375/845] Generating table/strings.h
ninja: build stopped: subcommand failed.
```

### Steps to reproduce

Try to compile.

--- a/src/lang/traditional_chinese.txt
+++ b/src/lang/traditional_chinese.txt
@@ -3308,7 +3308,7 @@ STR_MAPGEN_DATE_TOOLTIP                                         :{BLACK}選擇�
 STR_MAPGEN_NUMBER_OF_INDUSTRIES                                 :{BLACK}工業數量：
 STR_MAPGEN_NUMBER_OF_INDUSTRIES_TOOLTIP                         :{BLACK}選擇工業密度，或輸入自訂數字
 STR_MAPGEN_HEIGHTMAP_HEIGHT                                     :{BLACK}最高峰：
-STR_MAPGEN_HEIGHTMAP_HEIGHT_TOOLTIP                             :{BLACK}選擇遊戲版圖內最高峰的海拔，​​以海平面為基準計算
+STR_MAPGEN_HEIGHTMAP_HEIGHT_TOOLTIP                             :{BLACK}選擇遊戲版圖內最高峰的海拔，以海平面為基準計算
 STR_MAPGEN_HEIGHTMAP_HEIGHT_UP                                  :{BLACK}將地圖上最高峰的最大高度增加1
 STR_MAPGEN_HEIGHTMAP_HEIGHT_DOWN                                :{BLACK}將地圖上最高峰的最大高度減少1
 STR_MAPGEN_SNOW_COVERAGE                                        :{BLACK}雪地覆蓋率:

